SecondSavings/urls.py

```diff
@@ -31,6 +31,7 @@
     # Django admin
     path("admin/", admin.site.urls),
     # Django-allauth routes for authentication
+    path('TimeTracker/', include('TimeTracker.urls')),
```

Comment:
This will cause conflict:
WARNINGS:
?: (urls.W005) URL namespace 'TimeTracker' isn't unique. You may not be able to reverse all URLs in this namespace